src/Klimakoffer.jl

```diff
@@ -24,5 +24,7 @@ include("mesh.jl")
 include("model.jl")
 include("discretization.jl")
 include("numerics.jl")
-
+include("interpolation.jl") 
+include("imageprocessing.jl")
+include("PlotTutorial.jl")
```

Comment:
Where is the file PlotTutorial.jl? And, do we need it?
```suggestion
```

Reply:
If all that files under 'examples/test_instances/*' are not needed, what are my reference data for the tests?

Reply:
Well, then it might be a good practice to call it 'examples/reference_data' or similar. 'test_instances' suggest runtime data.
Here a cite from wikipedia: "[...] an instance is a concrete occurrence of any object, existing usually during the runtime of a computer program. Formally, "instance" is synonymous with "object" as they are each a particular value (realization), and these may be called an instance object; "instance" emphasizes the distinct identity of the object. The creation of an instance is called instantiation."

Reply:
done